lib/dependencies/DepBlockHelpers.js

```diff
@@ -33,7 +33,7 @@ DepBlockHelpers.getDepBlockPromise = function(depBlock, outputOptions, requestSh
 				"])";
 		}
 	}
-	return "Promise.resolve()";
+	return "new Promise(function(resolve) { resolve(); })";
```

Comment:
I don't know if it's true but 100% spec-compliant have it, MDN documents it, it smells like a standard you can safely count on ;) plus it's way faster than new Promise in all native implementations right now afaik